src/main/java/step1/Number.java

```diff
@@ -0,0 +1,60 @@
+package step1;
+
+import java.util.Objects;
+
+public class Number {
+    private final int number;
+
+    public Number(String formula) {
+        this.number = parserInt(formula);
+        validationCheck(this.number);
+    }
+
+    public int parserInt(String formula) {
+        if (isBlankOrNull(formula)) {
+            return 0;
+        }
+        return convertStringToNumber(formula);
+    }
+
+    public int convertStringToNumber(String formula) {
+        int result;
+        try {
+            result = Integer.parseInt(formula);
+        } catch (NumberFormatException e) {
+            throw new RuntimeException("숫자이외의 문자는 입력할 수 없습니다.");
+        }
+        return result;
+    }

```

Comment:
```suggestion
    public int convertStringToNumber(String formula) {
        try {
            return Integer.parseInt(formula);
        } catch (NumberFormatException e) {
            throw new RuntimeException("숫자이외의 문자는 입력할 수 없습니다.");
        }
    }
```
바로 return 해줘도 좋을 것 같아요! 😄

Reply:
넵!